website/templates/project/settings.mako

```diff
@@ -531,6 +462,74 @@
 
         % endif  ## End Retract Registration
 
+        % if enable_institutions:
+             <div class="panel panel-default scripted" id="institutionSettings">
+                 <span id="configureInstitutionAnchor" class="anchor"></span>
+                 <div class="panel-heading clearfix">
+                     <h3 class="panel-title">Project Affiliation / Branding</h3>
+                 </div>
+                 <div class="panel-body">
+                     <div class="help-block">
+                         % if 'write' not in user['permissions']:
```

Comment:
FYI, settings option in the nav only shows for admins on registrations right now: https://github.com/CenterForOpenScience/osf.io/blob/develop/website/templates/project/project_header.mako#L75

Reply:
Thank you for noticing that!